make/modules/java.desktop/lib/ClientLibraries.gmk

```diff
@@ -67,8 +67,10 @@ ifeq ($(USE_EXTERNAL_LCMS), true)
   # If we're using an external library, we can't include our own SRC path
   # as includes, instead the system headers should be used.
   LIBLCMS_HEADERS_FROM_SRC := false
-  # FIXME: Keep old behavior and reset LCMS_CFLAGS. This is likely a bug.
-  LCMS_CFLAGS :=
+  ifeq ($(call isTargetOs, bsd), false)
```

Comment:
This bug is being dealt with in https://github.com/openjdk/jdk/pull/24131. Thanks for reminding me about it.

Reply:
@magicus Yep, I saw that. I thought (hoped) that would be the proper fix, but didn't want to change anything for other platforms in this PR. :) I'll be merging in the remaining changes to the mainline, and update the PR within the week.